composer.json

```
@@ -1,6 +1,6 @@
 {
     "name": "intercom/intercom-php",
-    "description": "Intercom API client built on top of Guzzle 6",
+    "description": "Intercom API client built on top of Guzzle",
```

Good catch! Actually, now this library is built on top of HTTPlug, so what do you think about this? 😄 

```suggestion
    "description": "Intercom API client built on top of HTTPlug",
```

---

Done